public-ui/src/app/shared/components/breadcrumb/breadcrumb.component.ts

```diff
@@ -92,31 +99,31 @@ export class BreadcrumbComponent implements OnInit, OnDestroy {
    * Returns array of Breadcrumb objects that represent the breadcrumb trail.
    * Derived from current route in conjunction with the overall route structure.
    */
-  private buildBreadcrumbs(route: ActivatedRoute,
-    url: string = '',
-    breadcrumbs: Breadcrumb[] = []): Array<Breadcrumb> {
+  private buildBreadcrumbs(route: ActivatedRoute, breadcrumbUrl: string = '', breadcrumbs: Breadcrumb[] = []): Array<Breadcrumb> {
```

Comment:
Renamed `url` -> `breadcrumbUrl` so it won't conflict with the `url` pulled from `snapshot` below